src/develop/develop.c

```diff
@@ -1310,6 +1311,18 @@ void dt_dev_write_history(dt_develop_t *dev)
   dt_dev_write_history_ext(dev, dev->image_storage.id);
 }
 
+static int _dev_get_module_nb_records()
+{
+  sqlite3_stmt *stmt;
+  DT_DEBUG_SQLITE3_PREPARE_V2(dt_database_get(darktable.db),
+                              "SELECT count (*) FROM  memory.history",
```

Comment:
should be COUNT(*)

I would move this into history.h and use it in many places (at least in history.c, develop.c and lightrroom.c) where we are using this exact SQL statement. Would be a nice refactoring.

Reply:
EDIT: sorry this is counting memory not main.history so forget about this.